extensions/amp-analytics/0.1/variables.js

```diff
@@ -305,13 +240,9 @@ export class VariableService {
    * @param {string} value
    * @return {!Promise<string>}
    */
-  hashFilter_(value) {
+  hashMacro_(value) {
     return Services.cryptoFor(this.win_).sha384Base64(value);
   }
-
-  isFilterExperimentOn_() {
```

Comment:
nit: `variable-filters` is not used anywhere, please remove from `experiments.js`

Reply:
done